patterns/roks-quickstart/main.tf

```diff
@@ -38,6 +38,7 @@ locals {
          "name": "workload-cluster",
          "resource_group": "workload-rg",
          "disable_outbound_traffic_protection": true,
+         "force_delete_storage": true
```

Comment:
@toddgiguere what do you think for quickstart, true or false? I would hate to think we blow away a cuatomers data that they may want, so kind of thinking false?

Reply:
@Khuzaima05 talked with Todd on this - I think its fine to leave this as true for quickstart, however we should document it in `patterns/roks-quickstart/README.md` and maybe also in `reference-architectures/deploy-arch-ibm-slz-ocp-quickstart.md` @SirSpidey ?

We need to say something like:

> The quickstart flavor by default will force the removal of any persistent storage associated with the cluster during cluster deletion.

Reply:
I think the tricky part is showing this information at the right time. When would this come up? Can you delete a cluster from the catalog DA? Is this just setting up the parameter so that when you do delete the cluster, the persistent storage is deleted too?

So I don't know where to put the info because I don't know where they delete the cluster. 

If we want to post a CYA message, then the pattern readme is certainly one place. Another is in the features of the flavor in the catalog.json file (see screenshot).

reference architecture: I'm not sure that the reference architecture has any place where the consequences of deleting the cluster is covered

Here's how I would rewrite Conall's suggestion

When you delete the VPC that is created by [_this pattern,_ (in the readme) | _this QuickStart variation,_ (in the features)] ...
the associated persistent storage is also removed. To retain persistent storage when you delete a cluster, deploy the standard pattern|variation instead.

<img width="256" alt="image" src="https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/assets/8194208/e0f9a02b-9b55-432f-8bec-c6e2462298d7">

Reply:
@SirSpidey Cluster deletion happens when user does an undeploy from Projects (or destroy in Schematics). In Quickstart we are not exposing the ability to toggle - we are going to force delete, so we just want to make that clear

Reply:
Thanks @ocofaigh. I think my comments still hold -- that there's no place for us to add the info when the user is undeploying in projects. So the features info for the flavor/variation and the readme are the less the desirable places that we have.